### PR TITLE
Remove use of clinician first_name and last_name in favor of name

### PR DESCRIPTION
**Pull request by luketlancaster**
- [x] Add the Clubhouse Story ID: [ch21384]

### CODE REVIEW

**Comment by luketlancaster** [comment]
I just did a search for `first_name` and `last_name` and yanked all those out. Lemme know if you think I may have missed anything

File: src/js/views/patients/actions/actions_views.js [hunk 1]
```diff
@@ -90,14 +90,7 @@ const OwnerComponent = Droplist.extend({
...
         className: 'actions__owner button-secondary--compact w-100',
-        template: hbs`{{far "user-circle"}}{{ short }}{{ first_name }} {{ lastInitial }}`,
-        templateContext() {
-          return {
-            lastInitial() {
-              return this.last_name && `${ this.last_name[0] }.`;
-            },
-          };
-        },
+        template: hbs`{{far "user-circle"}}{{ short }}{{ name }}`,
```
Comment (paulfalgout) [inline src/js/views/patients/actions/actions_views.js]: roles have names. so this looks like an issue to me.  either we'll need to split the template here or make a templateContext helper that returns the appropriate result based on the model type

File: src/js/views/patients/sidebar/action/action-sidebar-activity-views.js [hunk 2]
```diff
@@ -79,14 +79,11 @@ const ActivityView = View.extend({
...
     const program = this.model.getProgram();
-    const toClinician = _.trim(`${ clinician.get('first_name') } ${ clinician.get('last_name') }`);
-    const role = editor.getRole().get('name');
-    const name = _.trim(`${ editor.get('first_name') } ${ editor.get('last_name') }`);
 
     return {
-      name,
-      role,
-      to_clinician: toClinician,
+      name: _.trim(editor.get('name')),
```
Comment (paulfalgout) [inline src/js/views/patients/sidebar/action/action-sidebar-activity-views.js]: shouldn't need the trims that's kinda legacy stuff from when on mainapp you could potentially not have one of the names

File: test/fixtures/config/clinicians.js [hunk 3]
```diff
@@ -9,14 +9,9 @@ module.exports = {
...
 
-    const first = faker.name.firstName();
-    const last = faker.name.lastName();
-
     return {
       id: faker.random.uuid(),
-      first_name: first,
-      last_name: last,
-      name: `${ first } ${ last }`,
+      name: `${ faker.name.firstName() } ${ faker.name.lastName() }`,
```
Comment (paulfalgout) [inline test/fixtures/config/clinicians.js]: could be `faker.name.findName()`

**Review by paulfalgout — CHANGES_REQUESTED** [review]

File: test/integration/patients/patient/dashboard.js [hunk 4]
```diff
@@ -299,7 +299,7 @@ context('patient dashboard page', function() {
       .find('[data-owner-region]')
       .find('button')
       .should('be.disabled')
-      .should('contain', 'Clinician M.');
+      .should('contain', 'Clinician McTester');
```
Comment (paulfalgout) [inline test/integration/patients/patient/dashboard.js:302]: I updated these to be the full name.  I know they're not displayed entirely, but that's what's in there.

**Review by paulfalgout — APPROVED** [review]

**Pull request merged**